### PR TITLE
Add result weight=1.0 in all methods add_result

### LINKED ISSUE #10: Consider instance weight when updating evaluators
The method ```add_result``` should take into account the weight of instances.

Expected signature:

```python
def add_result(self, sample, prediction, weight=1.0):
    # Code
```

### PR DESCRIPTION
**Pull request by gilbertoolimpio**
Fixes #10 .

Changes proposed in this pull request:
- add parameter `weitht=1.0` in all methods `add_result` in `measure_collection.py`
- add a call method `check_weights(weight)`
- add documentation

@scikit-multiflow/scikit-multiflow-repo-admins

Checklist

- [x] Code complies with PEP-8 and is consistent with the framework.
- [x] Code is properly documented.
- [x] Tests are included for new functionality.
- [x] Travis CI build passes.
- [x] Test Coverage is maintained (threshold 0.2%)

### CODE REVIEW

**Comment by jacobmontiel** [comment]
Hello @gilbertoolimpio ,

The change in #10 includes the handling of different weights when calculating the metrics. This was not clear in the description of the issue (my bad). I will update it to give more details.

File: src/skmultiflow/metrics/measure_collection.py [hunk 1]
```diff
@@ -83,9 +83,9 @@ def add_result(self, y_true, y_pred, weight=1.0):
         self.sample_count += weight
```
Comment (jacobmontiel) [inline src/skmultiflow/metrics/measure_collection.py]:
```suggestion
        self.sample_count += 1
```

**Comment by gilbertoolimpio** [comment]
Oh! I'm sorry! I'll fix my code. Thank you!

**Comment by gilbertoolimpio** [comment]
@jacobmontiel 

Now all sample_cout is 1!

Thank you for the tips.

**Comment by gilbertoolimpio** [comment]
I'm sorry @jacobmontiel 

I fix my code! Just `ẀindowClassificationMeasurements` was modified!

If there's anything else, just talk! It's very good to be able to contribute to you. 

Thank's in advanced!

**Comment by jmread** [comment]
Regarding in multi-output prediction, there shouldn't make any difference at the moment, since the weight is per sample, and all the multi-output metrics in the framework are sample-based at the moment. Sometimes it's also possible to average across columns but this is not as data-stream friendly and is currently not implemented.

**Comment by jacobmontiel** [comment]
Thanks @gilbertoolimpio 

I will merge the PR.

**Pull request merged**